lib/handlebars/runtime.js

```diff
@@ -85,6 +85,23 @@ export function template(templateSpec, env) {
       return typeof current === 'function' ? current.call(context) : current;
     },
 
+    splat: function() {
+      return Utils.extend.apply(null, arguments);
+    },
+
+    splatArgs: function() {
+      let splatMap = arguments[arguments.length - 1];
```

Comment:
nit: Would the code be clearer here if we did `splatArgs(args, splatMap) {}` rather than trying to extract from the end of the args list? Alt: Lets pass `splatMap` as the first parameter so we can avoid the `.length - 1` gymnastics.